apple/internal/entitlement_rules.bzl

```diff
@@ -282,7 +282,7 @@ def _entitlements_impl(ctx):
         explicit_minimum_os = None,
         features = ctx.features,
         objc_fragment = ctx.fragments.objc,
-        platform_type_string = str(ctx.fragments.apple.single_arch_platform.platform_type),
+        platform_type_string = ctx.attr.platform_type,
```

Comment:
Offline we talked, but I think this bug might have been hidden by the transition, which you have disabled.

Reply:
Successfully reverted the transition patch.

Getting an error about a missing attribute when trying to use it within the app. Mentioning because it appears to be an issue with the `ios_dylib` rule that you have here: https://github.com/bazelbuild/rules_apple/pull/987

`no such attribute 'families' in 'ios_dylib' rule`

Will post again after successfully testing the transition revert.

Reply:
Nvm on the `families` attribute.

Was missing this: https://github.com/bazelbuild/rules_apple/pull/987/files#diff-4a7ae7309bfed96760d671ad00e5726a6d0c466112275402d9944abd295d7328R1109-R1115

Reply:
Not having signing issues on the sim without this change + reverting the transition. Will close today unless I hear that this is need for some reason.

Reply:
I think it might still be worth fixing, just that now we know the transition is probably covering up the bug.